controllers/sessions/index.js

```diff
@@ -47,6 +47,7 @@ router.get('/', (req, res) => {
             user_id: req.session.user_id,
             email: req.session.email,
             first_name: req.session.first_name,
+            username: req.session.username,
```

Comment:
Do you need to store all this data in your session? I'm not sure you'd want much more than the user_id stored here.

Reply:
We are pulling these data to use throughout the site - e.g. in this PR to auto populate the logged in user as User 1.

Reply:
I also don't think we actually use first name anywhere do we? We can probably remove that line, same with email other than when we validate?

Reply:
Yeap I think you're right, I looked through I don't think we are using those, mostly user ID and username, added a trello card to remove.